<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="nombreproyectoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Nombre del Proyecto:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombreproyectolineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="rutaLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Ruta del Proyecto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rutalineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rutaButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Seleccionar Ruta</string>
   </property>
  </widget>
  <widget class="QLabel" name="plantaLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Tipo de Planta:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="plantacomboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>Personalizado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ball and Beam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Telescopio</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ascensor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PÃ©ndulo Invertido</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="crearproyectoerrorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="crearButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
